contracts/PriceRiskModule.sol

```diff
@@ -370,59 +261,85 @@ contract PriceRiskModule is RiskModule, IPriceRiskModule {
    *                 negative if probability of higher price
    * @param cdf Array where cdf[i] = prob of price lower/higher than i% of current price
    */
-  function setCDF(int40 duration, uint256[PRICE_SLOTS] calldata cdf)
+  function setCDF(int40 duration, SlotPricing[PRICE_SLOTS] calldata cdf)
     external
     onlyComponentRole(PRICER_ROLE)
     whenNotPaused
   {
     require(duration != 0, "|duration| < 1");
-    _cdf[duration] = cdf;
+    for (uint256 i = 0; i < PRICE_SLOTS; i++) {
+      require(
+        cdf[i].lossProb == 0 ||
+          (cdf[i].jrCollRatio <= WadRayMath.WAD &&
+            cdf[i].collRatio <= WadRayMath.WAD &&
+            cdf[i].jrCollRatio <= cdf[i].collRatio),
+        "Validation: invalid collateralization ratios"
+      );
+      _cdf[duration][i] = cdf[i];
+    }
+    uint256 durationParam = duration < 0
+      ? uint256(type(uint40).max + uint256(uint40(-duration)))
+      : uint256(uint40(duration));
+    _parameterChanged(IAccessManager.GovernanceActions.rmFiller1, durationParam, false);
   }
 
   /**
-   * @dev Sets the tolerance for price age
-   * @param oracleTolerance_ The new tolerance in seconds.
+   * @dev Sets the minimum duration before a policy can be triggered
+   * @param minDuration_ The new minimum duration in seconds.
    */
-  function setOracleTolerance(uint40 oracleTolerance_)
+  function setMinDuration(uint40 minDuration_)
     external
     onlyComponentRole(ORACLE_ADMIN_ROLE)
     whenNotPaused
   {
-    _state.oracleTolerance = oracleTolerance_;
+    _state.minDuration = uint32(minDuration_);
```

Comment:
We can't do that, because oracleTolerance is not part or a requirement of IPriceOracle. Also, I don't think there's a relation between those parameters.